server/handler.go

```diff
@@ -210,8 +210,13 @@ func NewHandler(cfg *config.Config) (*Handler, error) {
 	if cfg.Analytics.Credentials != nil {
 		// Attempts to get an authorized http client with given analytics credentials
 		analyticsClient = clientAuthorizedByCredentials(cfg, "analytics", cfg.Analytics.Credentials)
-		// overwrite the internalAPI to the GCP managed host
-		internalAPI, _ = url.Parse(config.GCPExperienceBase)
+		// overwrite the internalAPI to the GCP managed host if not initialized yet
+		if internalAPI == nil {
+			internalAPI, err = url.Parse(config.GCPExperienceBase)
+			if err != nil {
```

Comment:
pointless check for a constant

Reply:
Yeah. Let me remove it again. Will need new approval then :)

Reply:
ok, sure :)